examples/step-37/step-37.cc

```diff
@@ -1158,9 +1158,35 @@ namespace Step37
   // The function that runs the program is very similar to the one in
   // step-16. We do few refinement steps in 3D compared to 2D, but that's
   // it.
+  //
+  // Before we run the program, output some information about the detected
+  // vectorization level. Note that you are likely not getting AVX supported
+  // unless you specify a compiler flag like "-march=native" during
+  // configuration of deal.II.
```

Comment:
We already have the last piece in `intro.dox`. Do we need to mention `march=native` again?